### ISSUE TITLE
teligram desktop is not accessible

### ISSUE THREAD

**Issue by varunkrishna123** [telegramdesktop/tdesktop#8236]
hi teligram teem.
the teligram desktop is not accessible with screenreader  for blindd person 
i hop you will fix this soon

**Comment by Aokromes** [comment]
https://github.com/telegramdesktop/tdesktop/issues/476
